jikkou/kafka-connectors-values.yaml

```diff
@@ -271,7 +271,10 @@ apps:
         collectionName: CmRtcmMinimumDataEventAggregation
         generateTimestamp: true
         timestampField: eventGeneratedAt
-
+      - topicName: topic.CmPriorityPreemptionRequestEvent
```

Comment:
If we have a Kafka connector for this topic, shouldn't the mongo script also be updated to add the collection? Or are we relying on auto-creation through this connector?

Reply:
Added that to the mongo script.